### ISSUE TITLE
Unable to retire a package

### ISSUE THREAD

**Issue by Fale** [fedora-infra/pkgdb2#345]
Today I was trying to retire nf3d but something seems to not work (AKA error 500 from pkgdb APIs):

```
$ fedpkg retire "A core dependency has been retired"
rm '.gitignore'
rm 'nf3d.spec'
rm 'sources'
[master 7b83b41] A core dependency has been retired
 4 files changed, 1 insertion(+), 56 deletions(-)
 delete mode 100644 .gitignore
 create mode 100644 dead.package
 delete mode 100644 nf3d.spec
 delete mode 100644 sources
Counting objects: 3, done.
Delta compression using up to 4 threads.
Compressing objects: 100% (1/1), done.
Writing objects: 100% (3/3), 303 bytes | 0 bytes/s, done.
Total 3 (delta 0), reused 0 (delta 0)
remote: Emitting a message to the fedmsg bus.
remote: * Publishing information for 1 commits
To ssh://fale@pkgs.fedoraproject.org/rpms/nf3d
   9849166..7b83b41  master -> master
FAS password for user fale: 
Could not retire package: HTTPSConnectionPool(host='admin.fedoraproject.org', port=443): Max retries exceeded with url: /pkgdb//api/package/retire/ (Caused by ResponseError('too many 500 error responses',))
```

**Comment by pypingou** [comment]
So the actual error/message is hidden, it's a bug in pkgdb-cli that's fixed in https://github.com/fedora-infra/packagedb-cli/pull/92
I will cut a release soon

**Comment by pypingou** [comment]
The fix has been deployed in the release 2.13 of packagedb-cli, so we can close this ticket now :)

Thanks for your report!

**Comment by Fale** [comment]
thanks pingou, I can confirm this problem was fixed :)
